addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -5228,7 +5227,7 @@ class SnippetsMenu extends Component {
                     });
                 } catch (e) {
                     if (e instanceof RPCError) {
-                        const message = escape(_t("Could not install module %s", moduleDisplayName));
```

Comment:
Note: notification service is using t-out, this is not necessary (it will double escape)